syft/frameworks/torch/federated/dataloader.py

```diff
@@ -28,22 +28,22 @@ def default_collate(batch):
         elem_type.__module__ == "numpy"
         and elem_type.__name__ != "str_"
         and elem_type.__name__ != "string_"
-    ):
+    ):  # pragma: no cover
```

Comment:
why are these all here?

Reply:
Before it is from pytorch - just kept it for generality purposes

Reply:
I mean the pragmas

Reply:
This is to avoid coverage taking this into account. We don't and won't have tests for this part in the middle term